src/views/tutorial-view/index.tsx

```diff
@@ -121,7 +122,9 @@ export default function TutorialView({
             url={getVideoUrl({ videoId: video.id, videoHost: video.videoHost })}
           />
         )}
-        <MDXRemote {...content} components={MDX_COMPONENTS} />
+        <DevDotContent>
+          <MDXRemote {...content} components={MDX_COMPONENTS} />
```

Comment:
Nice :) We can probably go back and remove some `!important`s from the tutorial meta now too. I'll make a follow-up ticket, no need to do that here.

Reply:
https://app.asana.com/0/1202001387788447/1202088246939065